### PR TITLE
Test/Fix for Issue91

### PR DESCRIPTION
**Pull request by Stefaminator**
Like promised before i wrote a test that will fail because of issue #91. Wrapping destroys the line because it splits an escape sequence.

I recommend a change in the wrapping-method. Let's use wordwrap https://www.php.net/manual/de/function.wordwrap.php with cut=false.

### CODE REVIEW

**Comment by Stefaminator** [comment]
This should be fixed now. Please take a look at it.
wordwrap is used now. So we never wrap in a middle of a word. This is the way like poedit does. The change makes the Multibyte-Test obsolete. But I implemented another mulitbyte test. Please see the testWrapping method.

File: tests/WriteTest.php [hunk 1]
```diff
@@ -71,36 +73,39 @@ public function testWritePlurals()
...
-        $fh = fopen($this->resourcesPath . 'temp.po', 'r');
-        fgets($fh); // ignore line 1
-        fgets($fh); // ignore line 2
-        $this->assertEquals("\"multibyte.translá\"\n", fgets($fh));
-        $this->assertEquals("\"tion.1\"\n", fgets($fh));
-
-        // Revert encoding to previous setting
-        mb_internal_encoding($mbEncoding);
-    }
+// This test fails now because of migrating the wrapString-Method to wordwrap.
```
Comment (raulferras) [inline tests/WriteTest.php]: You can just remove this method :)

File: src/PoCompiler.php [hunk 2]
```diff
@@ -267,23 +267,33 @@ protected function cleanExport($string)
         return "$quote$string$quote";
     }
 
+//    /**
```
Comment (raulferras) [inline src/PoCompiler.php]: Just remove this method, no need to keep it commented.

**Comment by Stefaminator** [comment]
Ok, i pushed the requested improvements and some other minor fixes of IDE warnings.

**Comment by Stefaminator** [comment]
Is it possible to merge this and build a new release to be able to include the fixed relase via composer?

**Comment by raulferras** [comment]
@Stefaminator, thanks for the PR.
I pushed a new tag: `5.1.7` which you should be able to download with composer now ;)

**Comment by Stefaminator** [comment]
Thank you for this useful project!

**Pull request merged**